# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1306438813 25200
#      Thu May 26 12:40:13 2011 -0700
# Branch stable
# Node ID abc71e3148ac886e8e86812b4e126154bb6f6e64
# Parent  e7b03b8662a27f945e5ad988453e3a101c1134c7
geornd.m: Fix typo in 2-input calling form preventing execution.

* geornd.m: Replace typo 'n' with input variable 'p'.

diff --git a/scripts/statistics/distributions/geornd.m b/scripts/statistics/distributions/geornd.m
--- a/scripts/statistics/distributions/geornd.m
+++ b/scripts/statistics/distributions/geornd.m
@@ -55,17 +55,17 @@ function rnd = geornd (p, r, c)
       error ("geornd: R must be a positive integer or vector");
     endif
 
     if (any (size (p) != 1)
         && ((length (size (p)) != length (sz)) || any (size (p) != sz)))
       error ("geornd: n must be scalar or of size SZ");
     endif
   elseif (nargin == 1)
-    sz = size(n);
+    sz = size(p);
   elseif (nargin != 1)
     print_usage ();
   endif
 
 
   if (isscalar (p))
     if (p < 0 || p > 1)
       rnd = NaN (sz);
